sacc/tracers.py

```diff
@@ -786,3 +786,319 @@ def from_tables(cls, table_list):
                                 quantity=quantity,
                                 metadata=metadata)
         return tracers
+
+
+class BinZTracer(BaseTracer, tracer_type="bin_z"):  # type: ignore
+    """A tracer for a single redshift bin."""
```

Comment:
Could you expand on the docstring to make clear how this differs from the tracer which includes an n(z).